pkg/grub/rootfs.cfg

```diff
@@ -203,6 +203,13 @@ function set_x86_64_baremetal {
          set_global dom0_platform_tweaks "$dom0_platform_tweaks pci=realloc=off"
       fi
    fi
+   if [ "$smb_vendor" = "Kontron America" ]; then
+      smbios -t 1 -s 5 --set smb_product
+      # switch ttyXRUSB2/3 to RS485 Half-Duplex mode
+      if regexp -- "Agora Gateway" "$smb_product"; then
+         set_global hv_platform_tweaks "$hv_platform_tweaks xr_usb_serial_common.mode=2h,3h"
```

Comment:
it should be 'dom0_platform_tweaks' since you are applying to Linux kernel....

Reply:
@rene   yes, I fixed that already